certified-connectors/Scrive eSign/apiProperties.json

```diff
@@ -1,64 +1,143 @@
+          },
+          "parameters": {
+            "username": {
+              "type": "securestring",
+              "description": "The Scrive account email.",
+              "displayName": "Email for Scrive account",
+              "tooltip": "Provide the email for the Scrive account",
+              "uiDefinition": {
+                "constraints": {
+                  "clearText": true,
+                  "required": "true",
+                  "tabIndex": 2
+                }
+              }
+            },
+            "password": {
+              "type": "securestring",
+              "description": "The Scrive account password.",
+              "displayName": "Password for Scrive account",
+              "tooltip": "Provide the password for the Scrive account",
+              "uiDefinition": {
+                "constraints": {
+                  "clearText": false,
+                  "required": "true",
+                  "tabIndex": 3
+                }
+              }
+            },
+            "customApiUrl": {
+              "type": "securestring",
+              "description": "Specify Scrive API host. Default is https://scrive.com.",
+              "displayName": "API Host URL",
+              "tooltip": "Provide the host URL for the API.",
+              "uiDefinition": {
+                "constraints": {
+                  "clearText": true,
+                  "required": "false",
+                  "tabIndex": 4
+                }
+              }
+            }
+          }
+        },
+        {
+          "name": "OAuth",
+          "uiDefinition": {
+            "displayname": "OAuth",
+            "description": "OAuth authentication."
+          },
+          "parameters": {
+            "token": {
+              "oAuthSettings": {
+                "clientId": "b656c82f946739f0_36551",
+                "identityProvider": "oauth2",
+                "redirectMode": "Global",
+                "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+                "customParameters": {
+                  "authorizationUrl": {
+                    "value": "https://power-automate-integration.scrive.com/oauth/authorization"
+                  },
+                  "refreshUrl": {
+                    "value": "https://power-automate-integration.scrive.com/oauth/refresh"
+                  },
+                  "tokenUrl": {
+                    "value": "https://power-automate-integration.scrive.com/oauth/token"
+                  }
+                },
+                "scopes": [
+                  "FULL_ACCESS"
+                ],
+                "properties": {
+                  "IsFirstParty": "False",
+                  "IsOnbehalfofLoginSupported": false
+                }
+              },
+              "type": "oauthSetting"
+            }
+          }
         }
-      }
+      ]
+    },
+    "publisher": "Scrive AB",
```

Comment:
@marcusdelang please add the stackOwner property 

    "stackOwner": "Name of the owner of the service or product."